finetuner/__init__.py

```diff
@@ -308,6 +308,69 @@ def get_token() -> str:
     return ft.get_token()
 
 
+def build_model(
+    name: str,
+    model_options: Dict[str, Any] = {},
```

Comment:
having a deafult empty dict is dangerous. You need to set it as Optional and to None by default. Then inside the function you can do `model_options or {}`